modules/local/sanger-tol/btk/main.nf

```diff
@@ -45,12 +45,12 @@ process SANGER_TOL_BTK {
     // e.g. uploading the entry iyTipFemo_PRIMARY (which is what the resulting dataset is named)
     // will be a blank entry, with iyTipFemo_PRIMARY_filtered being the correct name due to the input fasta
     // both could be blank because of this.
+    //
+    // -r $pipeline_version \\
     """
     mv $reference ${meta.id}.fasta
 
-    nextflow run sanger-tol/blobtoolkit \\
-        -r $pipeline_version \\
-        -c $blobtoolkit_config_file \\
+    nextflow run /lustre/scratch124/tol/teams/tola/users/dp24/blobtoolkit/main.nf \\
```

Comment:
Should this point to a custom path on disk, or should this be passed through some kind of ext.arg - something like the following in `production.config`:

```
withName: SANGER_TOL_BTK {
    ext.pipeline = "/path/to/pipeline/main.nf"
}
```

And then in the module, have:

```
def pipeline = task.ext.pipeline ?: "sanger-tol/blobtoolkit"
"""
nextflow run ${pipeline}
...
```

That way external users can still run the release.

Reply:
Oh that shouldn't have been added, that was a for me only thing.

But I agree with your point. Done.